### PR TITLE
Import app banner

### PR DESCRIPTION
**Pull request by jkerr321**
image

### CODE REVIEW

File: src/components/desktop-app-banner/main.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
+@import "o-colors/main";
+@import "o-icons/main";
+@import "o-typography/main";
+$teal: oColorsGetPaletteColor('teal');
```
Comment (tavvy) [inline src/components/desktop-app-banner/main.scss]: can be removed

File: bower.json [hunk 2]
```diff
@@ -35,7 +35,9 @@
     "o-colors": "^4.1.5",
     "o-typography": "^5.4.2",
     "o-share": "^6.0.0",
-    "o-grid": "^4.3.6"
+    "o-grid": "^4.3.6",
+    "o-forms": "^4.0.4",
```
Comment (tavvy) [inline bower.json]: I think `o-forms` can be removed. and maybe `o-icons` can be updated to latest version only?

File: manifest.js [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+		partial: 'bottom/lazy', // path to template
+		messageId: 'ssi-join-prompt'// id as specified in ammit-api
+	},
 	b2bUpsellBanner: {
 		partial: 'bottom/b2b-upsell',
 		messageId: 'b2c-invite-colleague-banner'
 	},
+	desktopAppBanner: {
+		partial: 'bottom/desktop-app-banner',
+		messageId: 'desktop-app-banner'
```
Comment (tavvy) [inline manifest.js]: can we change the messageId to `desktopAppBanner` please

File: main.scss [hunk 4]
```diff
@@ -7,3 +7,4 @@
 }
 
 @import './src/components/invite-colleagues/main';
+@import './src/components/desktop-app-banner/main';
```
Comment (tavvy) [inline main.scss]: no longer needed

File: src/components/desktop-app-banner/main.scss [hunk 5]
```diff
@@ -0,0 +1,42 @@
+$white: oColorsGetPaletteColor('white');
```
Comment (tavvy) [inline src/components/desktop-app-banner/main.scss]: need to explicitly import `o-icons`, `o-colors` and `o-typography`

File: src/components/desktop-app-banner/main.scss [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+button {
+	.has-sent & {
+		display: none;
+	}
+}
+.n-app-banner__success {
+	@include oTypographySans($scale: -1);
+	// display: none;
```
Comment (tavvy) [inline src/components/desktop-app-banner/main.scss]: remove?

File: main.scss (hunk 4 above)
Comment (jkerr321) [reply main.scss]: disagree!

**Pull request merged**